app/lib/aether/aether.coffee

```diff
@@ -248,6 +250,13 @@ module.exports = class Aether
         ++count
     return count
 
+Aether.getTokenSource = (raw) ->
+  if /^\u56E7[a-zA-Z0-9+/=]+\f$/.test raw
```

Comment:
Its probably related to how we parse cpp code, but I am not sure. Can you explain what we are checking here, and how are we getting readable code from `raw`?

Reply:
explained offline